clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -360,6 +367,18 @@ void testSendUnsentRequest() {
         assertFalse(networkClient.hasAnyPendingRequests());
     }
 
+    @Test
+    void testMetadataErrorEvent() {
```

Comment:
for this test we are essentially testing if networkClientDelegate is enqueue an error event to the background event queue, so i don't think we need this test here since you've written tests in network client delegate and integration test.